web/config/initializers/shopify_app.rb

```diff
@@ -1,10 +1,34 @@
 # frozen_string_literal: true
 
 ShopifyApp.configure do |config|
+  config.webhooks = [
+    # After a store owner uninstalls your app, Shopify invokes the APP_UNINSTALLED webhook
+    # to let your app know.
+    { topic: "app/uninstalled", address: "api/webhooks/app_uninstalled" },
+
+    # NOTE 1: To register the URLs for the three GDPR topics that follow, please set the appropriate
```

Comment:
I wonder if we can add the GDPR webhooks to the config in a separate method. I like the comments but it'd be nice to not have them in the "main" body of the config file.